curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/66643c93e05093c728abdbe9.md

```diff
@@ -11,7 +11,7 @@ Before moving on, take a moment to review how functions work.
 
 Declare a function named `addTwoNumbers`. This function should take two arguments and return the sum of those two arguments.
 
-Then declare a `sum` variable and assign it the value of calling your `addTwoNumbers` function with `5` and `10` as the arguments. Log the `sum` variable to the console.
+Then declare a `sum` variable and assign it the value of calling your `addTwoNumbers` function with `5` and `10` as the arguments. Log the `sum` variable to the console. Refer to `Step 47`.
```

Comment:
Per discussion on the issue, we can't reference step numbers since they may change

Reply:
Removed the reference to step 47